src/common/config.cc

```diff
@@ -1218,7 +1218,11 @@ Option::value_t md_config_t::_expand_meta(
       } else if (var == "id") {
 	out += values.name.get_id();
       } else if (var == "pid") {
-	out += stringify(getpid());
+        char *_pid = getenv("PID");
+	if (_pid)
```

Comment:
might want to be consistent when using tab and spaces.

Reply:
Done.